src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -79,11 +82,15 @@ public void given_initiate_tasks_with_follow_up_overdue_category_then_cancel_tas
             taskIdDmnColumn
         );
 
+        waitSeconds(2);
+
         // Then cancel the task1
         String eventToCancelTask = "uploadHomeOfficeBundle";
         String previousStateToCancelTask = "awaitingRespondentEvidence";
         sendMessage(caseIdForTask1, eventToCancelTask, previousStateToCancelTask, "", false);
 
+        waitSeconds(2);
```

Comment:
This test alone will take 10 seconds to finish can we short  the waits I don't think we need that many

Reply:
just for safe side I had to do that because on aat when ASB is enabled, it has to consume from topic and could be slight latency issue. This can be revisited when we integrate with ASB spinning story.